test/sensor_simulator/sensor_simulator.cpp

```diff
@@ -30,6 +31,7 @@ void SensorSimulator::setSensorDataToDefault()
 	_flow.setRateHz(50);
 	_rng.setRateHz(30);
 	_vio.setRateHz(30);
+	_airspeed.setRateHz(30); // TODO: check this rate
```

Comment:
@kamilritz I think the rate is currently quite fast, like 100Hz.

Reply:
@dagar Is this [comment](https://github.com/PX4/Firmware/blob/master/src/drivers/differential_pressure/sdp3x/SDP3X.hpp#L66) correct?

Reply:
100 Hz is more or less correct, although the SDP3X can be much faster. All the others are currently sampled at 100 Hz, but I'll do a pass to review them all shortly. https://github.com/PX4/Firmware/pull/13586